scripts/build-wvr-components.js

```diff
@@ -17,9 +19,10 @@ const dirPath = `dist/bundle/${dirName}`;
   ];
 
   fs.ensureDir(dirPath);
+  fs.ensureDir(latestPath);
 
   await concat(files, `${dirPath}/weaver-components.js`);
+  await concat(files, `${latestPath}/weaver-components.js`);
```

Comment:
This is going to take 2x as long. We should concat once, and then copy the file, as is done for docs.